go/client/cmd_encrypt.go

```diff
@@ -0,0 +1,118 @@
+// Copyright 2015 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package client
+
+import (
+	"errors"
+
+	"golang.org/x/net/context"
+
+	"github.com/keybase/cli"
+	"github.com/keybase/client/go/libcmdline"
+	"github.com/keybase/client/go/libkb"
+	"github.com/keybase/client/go/protocol"
+	"github.com/keybase/go-framed-msgpack-rpc"
+)
+
+type CmdEncrypt struct {
+	libkb.Contextified
+	filter       UnixFilter
+	recipients   []string
+	trackOptions keybase1.TrackOptions
+}
+
+func NewCmdEncrypt(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.Command {
+	return cli.Command{
+		Name:         "encrypt",
+		ArgumentHelp: "<usernames...>",
+		Usage:        "Encrypt messages or files for keybase users",
+		Action: func(c *cli.Context) {
+			cl.ChooseCommand(&CmdEncrypt{
+				Contextified: libkb.NewContextified(g),
+			}, "encrypt", c)
+		},
+		Flags: []cli.Flag{
+			cli.BoolFlag{
+				Name:  "l, local",
+				Usage: "Only track locally, don't send a statement to the server.",
+			},
+			cli.BoolFlag{
+				Name:  "y",
+				Usage: "Approve remote tracking without prompting.",
+			},
+			cli.StringFlag{
+				Name:  "m, message",
+				Usage: "Provide the message on the command line.",
+			},
+			cli.StringFlag{
+				Name:  "i, infile",
+				Usage: "Specify an input file.",
+			},
+			cli.StringFlag{
+				Name:  "o, outfile",
+				Usage: "Specify an outfile (stdout by default).",
+			},
```

Comment:
Great, also maybe 2 different stories here.  We can iterate, they might not all be required for M2.